Octokit.Tests/Models/IssueTest.cs

```diff
@@ -257,7 +257,7 @@ public void CreatesAnIssueUpdateRequestObject()
 
             var update = issue.ToUpdate();
 
-            Assert.Null(update.Labels);
+            Assert.NotNull(update.Labels);
```

Comment:
@ryangribble it's all good, I just need to cast my mind back to the earlier discussions about this.

#718 is the original PR where I implementing this, and to TL;DR it:

 - initially, the result from `ToUpdate()` should have a value of `null` for `Labels` - this is how we say "don't change anything" to the server as part of the update, as `labels` won't be serialized
 - if you invoke `update.AddLabel("label")` we create a new list lazily, and this collection **replaces** whatever labels are previously set on the server
 - if you invoke `update.ClearLabels()` this creates an empty list, to indicate that all labels should be removed as part of the update 

It's might come across as a bit explicit, but I like the verbosity that it gives you in the code.

Anyway, coming back to this change. It looks like we're populating the `Labels` array from what exists in the issue, and while that's similar to the original intent I didn't want users to be manipulating the `Labels` collection directly to remove labels - I wanted to see them use the first-class `AddLabel` and `ClearLabel` methods on `IssueUpdate`.